packages/openneuro-components/src/dropdown/DropdownList.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { Dropdown } from './Dropdown'
+import './dropdown.scss'
+
+export interface DropdownListProps {
+  items: [
+    {
+      label: string
+      icon: string
+      onClick: () => void
+      active: boolean
+    },
+  ]
+}
+
+export const DropdownList = ({ items }: DropdownListProps) => {
+  return (
+    <Dropdown
+      label={
+        <div className="list-label">
+          <b>SORT BY:</b> Newest{' '}
+          <i className="fas fa-exchange-alt fa-rotate-90" />{' '}
+        </div>
+      }>
+      <div className="dropdown-list">
+        <ul>
```

Comment:
Is there a reason why we're using `ul` and `li` elements instead of `select` and `option` elements?

Reply:
Select and option are real limited when styling. This is pretty standard.